pubtools/exodus/gateway.py

```diff
@@ -66,7 +66,10 @@ def unpack_response(self, response):
             except:
                 raise outer
 
-            LOG.error("unsuccessful response from exodus-gw: %s", body)
+            LOG.error(
+                "Unsuccessful response from exodus-gw: %s",
+                body.get("detail", self),
```

Comment:
I'm not getting the 'self' as default value here.  If body doesn't contain detail, how does it make sense to use stringified ExodusGatewaySession as the message?  Did you perhaps mean 'response' or 'response.content' here?

Reply:
Yeah, I think the aim was to default to `body`, assuming `self` would refer to the dict using `.get()`. Probably incorrect. I've changed it to `body.get("detail") or body`.